daemon/src/engine/server/sv_ccmds.cpp

```diff
@@ -369,6 +385,48 @@ static void SV_Systeminfo_f()
 	Info_Print( Cvar_InfoString( CVAR_SYSTEMINFO, false ) );
 }
 
+class ListMapsCmd: public Cmd::StaticCmd
```

Comment:
There's an #if that only enables it on a dedicated server.
The other definition is on cgame.
They have  slightly different goals, the cgame one tries to have a pretty format for humans (formatting looks a bit broken on that btw. While this new one is mainly to be machine readable.